app/Http/Controllers/V2/Nurseries/CreateNurseryWithFormController.php

```diff
@@ -28,6 +31,25 @@ public function __invoke(CreateEntityFormRequest $formRequest)
             'status' => EntityStatusStateMachine::STARTED,
         ]);
 
+        $lastTask = Task::where('project_id', $project->id)
+            ->orderBy('created_at', 'desc')
+            ->first();
+
+        if ($lastTask) {
+            $nextReportDueDate = Carbon::parse($lastTask->due_at)->addWeeks(4);
+
+            if (Carbon::now()->lessThan($nextReportDueDate)) {
+                NurseryReport::create([
+                    'framework_key' => $lastTask->project->framework_key,
+                    'task_id' => $lastTask->id,
+                    'nursery_id' => $nursery->id,
+                    'status' => 'due',
+                    'due_at' => $lastTask->due_at,
+                ]);
```

Comment:
This would be better done through the associations on the task:
```
$lastTask->nurseryReports()->create([
    'framework_key' => $project->framework_key,
    'nursery_id' => $nursery->id,
    'status' => 'due',
    'due_at' => $lastTask->due_at,
]);
```